src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Services/SyntaxFacts/ExternalSourceInfo.cs

```diff
@@ -3,7 +3,7 @@
 // See the LICENSE file in the project root for more information.
 
 
-namespace Microsoft.CodeAnalysis.LanguageServices
+namespace Microsoft.CodeAnalysis
```

Comment:
i'm not a huge fan of these moving into the top level namespace.  that's a pretty important one and it feels icky to pollute it.

I would be fine with a general location for extensions/shared/utilities to all get dumped into.

Reply:
Sure, I can move this into a separate namespace. Does it seem fine to have `ISyntaxFacts` be in `Microsoft.CodeAnalysis`?

Reply:
i would prefer not.  it's still too much of a grab-bag.  i'd almost argue that nothing should be there except public types.

Reply:
Sounds good. Let me move all of this to `Microsoft.CodeAnalysis.SyntaxFacts`

Reply:
👍

Reply:
Actually attempting to move everything to `Microsoft.CodeAnalysis.SyntaxFacts` causes tons of files changes to import the new namespace. I think I am just going to switch back to `Microsoft.CodeAnalysis.LanguageServices` to keep the changes minimal and still address the core issue.